utils/startCamera.js

```diff
@@ -0,0 +1,11 @@
+import { openCamera } from "../redux/startCameraSlice";
+import { BarCodeScanner } from "expo-barcode-scanner";
+
+export const __startCamera = async (dispatch) => {
```

Comment:
Should it have a double underscore as a function name?

Reply:
That usually signifies that it's an internal function that should not be used outside of this project.